testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite6NonGeneric.java

```diff
@@ -59,9 +58,6 @@ public List<SuiteTestRun> getTestRuns(EnvironmentConfig config)
                 testOnEnvironment(EnvMultinodeKafkaSaslPlaintext.class)
                         .withGroups("configured_features", "kafka")
                         .build(),
-                testOnEnvironment(EnvMultinodePhoenix4.class)
```

Comment:
Could you remove `phoenix` from `EnvMultinodeAllConnectors`?

Reply:
Done